phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -976,7 +976,7 @@ public function format_fieldsynopsis_modifier_text($value, $tag) {
 
     public function format_methodsynopsis($open, $name, $attrs) {
         if ($open) {
-            $this->params = array("count" => 0, "opt" => 0, "content" => "");
+            $this->params = array("count" => 0, "opt" => 0, "content" => "", "repeatable" => '');
```

Comment:
Mismatch between use of `"repeatable"` here an `"ellipsis"` below.

Reply:
Thanks!